app/src/main/java/com/willowtree/vocable/room/VocableDatabase.kt

```diff
@@ -20,11 +25,52 @@ abstract class VocableDatabase : RoomDatabase() {
             }
         }
 
+        private val MIGRATION_2_3: Migration = object : Migration(2, 3) {
```

Comment:
Do you have the time to write unit tests around this migration?  This all looks good to me, but some tests would make me feel better.

Reply:
Yeah, the only room migration I have experience with is the default 'drop everything'.

Reply:
Just added some tests for this migration!